app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    element = command.split(" ")
```

Comment:
Consider using more descriptive variable names. For example, instead of 'element', you could use 'command_parts' to make it clear that this variable holds parts of the command. This aligns with the checklist's recommendation to use descriptive and correct variable names.